packages/tools/kadena-cli/src/prompts/account.ts

```diff
@@ -1,36 +1,36 @@
-// import { select } from '@inquirer/prompts';
-// import { program } from 'commander';
+import { input, select } from '@inquirer/prompts';
+import type { IPrompt } from '../utils/createOption.js';
 
-// export async function accountSelectPrompt(
-//   isOptional: boolean = false,
-// ): Promise<string | 'skip'> {
-//   const existingAccounts: ICustomAccountChoice[] = await getExistingAccounts();
+export const publicKeysPrompt: IPrompt = async () =>
+  await input({
+    message: 'Enter zero or more public keys (comma separated).',
+  });
 
-//   let choices: ICustomAccountChoice[] = existingAccounts.map((account) => ({
-//     value: account.value,
-//     name: account.name,
-//   }));
+export const accountNamePrompt: IPrompt = async () =>
+  await input({
+    message: 'Enter an account name.',
+  });
 
-//   if (isOptional) {
-//     choices.push({
-//       value: 'skip',
-//       name: 'Account is optional. Continue to next step',
-//     });
-//   }
+export const amountPrompt: IPrompt = async () =>
+  await input({
+    validate(value) {
+      return !isNaN(parseFloat(value));
```

Comment:
I usually prefer to add a `.replace(',', '.')` when using parseFloat() on user input so that it accepts both.